src/helpers.js

```diff
@@ -1,5 +1,6 @@
 import {isJSON} from "./utils";
 import {eventData} from "./methods";
+import {ATTRS} from '@/configs'
```

Comment:
Should be `import {ATTRS} from './configs'` as the `@` will cause a bug when building production files.